rebasehelper/cli.py

```diff
@@ -123,6 +124,10 @@ def run():
             else:
                 main_handler.setLevel(logging.DEBUG)
             ConsoleHelper.use_colors = ConsoleHelper.should_use_colors(config)
+            if config.bugzilla_id:
+                name, version = BugzillaHelper.prepare_rebase_repository(config.bugzilla_id)
+                os.chdir(name)
+                config.sources = version
```

Comment:
Why not assign `config.sources` in place, there is no need for extra variable. Also, I think `component` would be better name than `name`. Or perhaps `package` or `package_name`, if you want to avoid the word component for some reason.